Library/ViewModels/LandingPageViewModel.swift

```diff
@@ -22,15 +24,27 @@ public final class LandingPageViewModel: LandingPageViewModelType, LandingPageVi
       .map(cards)
       .skipNil()
 
+    self.dismissViewController = self.ctaButtonTappedSignal
```

Comment:
If the user closes out of the app without closing the landing page, the user defaults value will never be set, so they'll see the landing page again the next time they open the app. I'm not sure this is the desired behavior - we might want to immediately set the user defaults value as soon as the landing page screen loads. You might want to confirm with product on this.

Reply:
Yes, I thought about that while developing. I'd decided to set the values on the button action to make sure that the users saw and interacted with the page (tapped the button). I will check with product, though.

Reply:
After standup today, we've decided to set `hasSeenLandingPage = true` when the view is presented.